apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -53,6 +53,7 @@ visualVm=""
 targetDirectory=
 dbUrl="jdbc:bfd-test:hsqldb:mem"
 v2Enabled="true"
+preAdjEnabled="true"
```

Comment:
Wouldn't defaulting to true in this script cause it to be enabled in prod on the next deployment once this is merged into master?

Reply:
Never mind this is just for local tests.